presto-docs/src/main/sphinx/client.rst

```diff
@@ -0,0 +1,26 @@
+*******
+Clients
+*******
+
+A client is used to send queries to Presto and receive results, or otherwise
+interact with Presto and the connected data sources.
+
+The client, such as the :doc:`command line interface </installation/cli>`, can provide
+a user interface directly. Clients like the :doc:`JDBC driver </installation/jdbc>`,
+provide a mechanism for other tools to connect to Presto.
+
+The following clients are available:
+
+.. toctree::
+    :maxdepth: 1
+
+    installation/cli
+    installation/jdbc
+
+In addition, the community provides numerous other clients for platforms such as
```

Comment:
Let's change this so that "numerous other clients" is the link.

Reply:
done